build/parent-pom/pom.xml

```diff
@@ -121,11 +124,8 @@
                     <reportsDirectory>${project.build.directory}/surefire-reports</reportsDirectory>
                     <junitxml>.</junitxml>
                     <filereports>WDF TestSuite.txt</filereports>
-                    <argLine>
-                        --add-opens=java.base/java.util=ALL-UNNAMED
-                        --add-opens=java.base/sun.net.www.protocol.jar=ALL-UNNAMED
-                        --add-exports=java.base/sun.nio.ch=ALL-UNNAMED
-                    </argLine>
+                    <!-- ‘scalatest.argLine‘ defined in root-pom using value of ‘default.scalatest.argLine‘ -->
+                    <argLine>${scalatest.argLine}</argLine>
```

Comment:
Can't we simply remove `<argLine>` here and allow it to be inherited from the `root-pom`?

Reply:
This plugin is not present on parent root-pom. So nothing pre-defined.
There was an update to used `default.scalatest.argLine`. This usage is correct as it is defined in current project.

Reply:
Improved in commit c97743db1e60b6efb4ce03c96e9b761dccce40aa.